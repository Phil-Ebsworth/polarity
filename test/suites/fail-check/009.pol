data Foo { }
codata Bar(n: Foo) { }
codef Baz(n : Type, l1:Bar(n)) : Bar(n) { }
