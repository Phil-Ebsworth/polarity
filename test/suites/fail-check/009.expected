Cannot automatically decide whether Type and Foo unify